.github/workflows/percy_snapshots.yml

```diff
@@ -53,5 +55,6 @@ jobs:
         run: ${{ steps.vars.outputs.merged_percy_script }}
         env:
           PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
+          REACT_APP_MAPBOX_API_KEY: ${ secrets.REACT_APP_MAPBOX_API_KEY }}
```

Comment:
nit: for organizational purposes, can this line be either the first or last of the `env` variables so it doesn't break up the other Percy ones?